README.md

```diff
@@ -1,50 +1,53 @@
-Emoji Support
-====
+[![Gluon](.github/assets/gluon_logo.svg)](https://gluonhq.com)
+
+[![Build](https://github.com/gluonhq/emoji/actions/workflows/build.yml/badge.svg)](https://github.com/gluonhq/emoji/actions/workflows/build.yml)
+[![Maven Central](https://img.shields.io/maven-central/v/com.gluonhq/emoji)](https://search.maven.org/#search|ga|1|com.gluonhq.emoji)
+[![License](https://img.shields.io/github/license/gluonhq/emoji)](https://opensource.org/licenses/GPL-3.0)
 
 The GluonFX Emoji component provides Emoji support to JavaFX applications.
 
-Emoji
----------
+## Projects
+
+### Emoji
 Raw emoji support, no JavaFX control
 
-Emoji-Samples
----------
+### Emoji-Samples
 Simple JavaFX application that displays all emojis
 
-Emoji-Updater
----------
+### Emoji-Updater
 Utility to manually update Emoji to the latest emoji data, whenever a new version is released
 
-Build
-=====
+## Build
 
-To update the emoji list:
+Build the emoji artifact:
 
 ```
-cd emoji-updater
-mvn javafx:run
-cd ..
+mvn clean install -f emoji
 ```
 
-To build the Emoji artifact:
+Use the emoji dependency in a Maven project:
 
 ```
-cd emoji
-mvn clean install
-cd ..
+<dependency>
+    <groupId>com.gluonhq</groupId>
+    <artifactId>emoji</artifactId>
+    <version>${version}</version>
+</dependency>
 ```
 
-To run the sample application:
+Run the sample application:
+
+```
+mvn javafx:run -f samples
+```
 
+Update the emoji list:
```

Comment:
In theory this is only needed when there is a change in the original emoji.json, but when done, it should be called _before_ building the emoji component, which will use the new cvs and png files.

Reply:
Do you want me to create a separate section on how to update emojis via `emoji.json`?

Reply:
The updater project grabs the latest emoji.json (from a given [commit](https://github.com/gluonhq/emoji/blob/main/emoji-updater/src/main/java/com/gluonhq/emoji/tools/Main.java#L39), that is), parses it to emoji.csv and puts this file and the sprite sheet pngs into emoji project.
Therefore, updating the emoji project _after_ building it doesn't make sense. If developers need this, they should do it before.
So my point was that the order in the readme should be:
- update (only if needed, when there is a new emoji version, for instance)
- emoji
- sample (just to test)

Reply:
Right. Running updater is neither mandatory nor will be used by every user/developer of the library. Therefore, I feel the instructions on how to use it is less important.

Reply:
Okay, but users don't know that. Just add something like: "In case you need to update the Emoji project, run...

Reply:
Do you feel that updater needs to have its own readme where we describe what it does and how can someone update the emoji data source?

Reply:
yes, that makes sense